### PR TITLE
Enable code coverage report on Linux

### PR DESCRIPTION
**Pull request by sfc-gh-ext-simba-hx**
https://github.com/snowflakedb/snowflake-sdks-drivers-issues-teamwork/issues/115
DO NOT MERGE until January release finished.
The build coverage number can be found at the end of the test on linux with php 8.1: https://github.com/snowflakedb/pdo_snowflake/actions/runs/3972843116/jobs/6811084577#step:6:198

### CODE REVIEW

File: .github/workflows/build_test_linux_php8-1.yml [hunk 1]
```diff
@@ -46,6 +47,8 @@ jobs:
         #  run: |
         #    sudo apt-get install valgrind
 
+        - name: Setup LCOV
+          uses: hrishikesh-kadam/setup-lcov@v1
```
Comment (sfc-gh-igarish) [inline .github/workflows/build_test_linux_php8-1.yml:51]: why using hrishikesh-kadam? Is it ok to use? any license requirement?

Comment (sfc-gh-ext-simba-hx) [reply .github/workflows/build_test_linux_php8-1.yml]: removed

**Review by sfc-gh-igarish — APPROVED** [review]
LGTM

**Comment by sfc-gh-ext-simba-hx** [comment]
Close as the actual change is merged in #325

**Pull request closed without merging**